<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mWindows</class>
 <widget class="QMainWindow" name="mWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Postulante S.A.</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblDate">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblHour">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Hour</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwCharacters">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>271</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRequest">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>REQUEST</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
